crank/DihedralScanner.py

```diff
@@ -434,8 +434,8 @@ def finish(self):
 
     def draw_ascii_image(self):
         """ Return a string with ASCII colors showing current running status """
-        if not hasattr(self, 'grid_energies') or not hasattr(self, 'running_job_path_info'):
-            return "draw_ascii_image failed: grid_energies or running_job_path_info not available"
+        if not hasattr(self, 'grid_energies') or not hasattr(self, 'opt_queue'):
+            return "draw_ascii_image failed: grid_energies not available"
```

Comment:
It would be great if this line is also updated to:
`return "draw_ascii_image failed: grid_energies or opt_queue not available"`